app/views/comments/_form.html.erb

```diff
@@ -45,24 +59,20 @@
         <% end %>
         <textarea aria-label="Comment Text" style="border: 1px solid #bbb;border-bottom-left-radius: 0;border-bottom-right-radius: 0;border-bottom: 0;padding: 10px;" onFocus="editing=true" name="body" class="form-control" id="text-input" rows="6" cols="40" placeholder="<%= placeholder %>"><%= body %></textarea>
         <div id="imagebar">
-
           <div id="create_progress" style="display:none;" class="progress float-right">
             <div id="create_progress-bar" class="progress-bar progress-bar-striped progress-bar-animated" style="width: 0%;"></div>
           </div>
-
           <p>
             <span id="create_uploading" class="uploading" style="display:none;">
               <%= translation('comments._form.uploading') %>
             </span>
             <span id="create_prompt" class="prompt">
-
               <span style="padding-right:4px;float:left;" class="d-none d-md-inline">
                 <%= raw translation('comments._form.drag_and_drop') %>
               </span>
-
               <!-- http://stackoverflow.com/questions/11235206/twitter-bootstrap-form-file-element-upload-button -->
-              <label for="fileinput">
-                <input id="fileinput" type="file" name="image[photo]" style="display:none;" />
+              <label for="fileinput-choose-one<%= dropzone_large_id %>">
+                <input id="fileinput-choose-one<%= dropzone_large_id %>" type="file" name="image[photo]" style="display:none;" />
```

Comment:
Will this change styling at all? Noting that sometimes adding unique classnames instead of ids can be better since elements are allowed to have more than one class but only one ID. I tend to use classes instead of IDs for that reason, but it's just a convention or habit, so no big deal here, just curious!

Reply:
Oh, that's a really good point about styling. I did a simple search in VSCode for `#fileinput` and `input#fileinput` but it didn't come up with anything in `/assets/stylesheets`, so I think this is good. It looks as normal when I load it up in development too.

That's also a good point about classes vs. IDs. Since my ultimate goal is to be able to use `e.target` to set `$D.selected` upon file upload, I thought an ID made the most sense here... Good reminder though that I can use classes in other situations.

I was aware of the one ID per page rule. `dropzone_large_id` includes the comment ID, so it's going to resolve to a unique ID every time this form is rendered (`#fileinput-reply-large-22` or something like that)... Which means that there shouldn't be any conflicting IDs here, at least throughout `comments-list`!